cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -754,7 +753,7 @@ func TestGetControllerManagerCommand(t *testing.T) {
 				},
 				CertificatesDir:   testCertsDir,
 				KubernetesVersion: cpVersion,
-				FeatureGates:      map[string]bool{string(features.IPv6DualStack): true},
+				FeatureGates:      map[string]bool{"IPv6DualStack": true},
```

Comment:
please use this constant instead of the literal
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/features/features.go#L32
[1]